lib/wovnrb/headers.rb

```diff
@@ -203,5 +207,21 @@ def dirname
         '/'
       end
     end
+
+    def search_engine_bot?
+      return false unless @env.key?('HTTP_USER_AGENT')
+
+      bots = %w[Googlebot/ bingbot/ YandexBot/ YandexWebmaster/ DuckDuckBot-Https/ Baiduspider/ Slurp Yahoo]
+      bots.each do |bot|
+        return true if @env['HTTP_USER_AGENT'].include?(bot)
+      end
+      false
```

Comment:
this looks like it could be `return bots.any { |bot| @env['HTTP_USER_AGENT'].include?(bot) }`

Reply:
Yep, that's much cleaner.